static/js/tools/choropleth/choropleth.tsx

```diff
@@ -333,13 +389,47 @@ class ChoroplethMap extends Component {
           .attr("font-weight", "bold")
           .text(title)
       );
+
+      const dateComponent = 
+    
+      ReactDOM.render(this.datePicker(), document.getElementById('date'));
+
```

Comment:
I guess that TypeScript does not care about whitespace, but humans might find the extra blank lines confusing.

Reply:
fixed.